astropy/table/index.py

```diff
@@ -825,6 +841,51 @@ def __getitem__(self, item):
         return self.table[rows]
 
 
+class TableLocIndices(TableLoc):
+    def __init__(self, table):
+        super().__init__(table)
```

Comment:
The `__init__` does not need to be explicitly defined here.

Reply:
the TableILoc class had one so i added  it..should i remove it then?

Reply:
Yes, I think so.